avro-builder/builder/src/main/java/com/linkedin/avroutil1/builder/operations/codegen/own/AvroUtilCodeGenOp.java

```diff
@@ -172,7 +172,9 @@ public void run(OperationContext opContext) throws Exception {
       } catch (Exception e) {
         errorCount++;
         //TODO - error-out
-        LOGGER.error("failed to generate class for " + fullname + " defined in " + fileParseResult.getContext().getUri(), e);
+        System.err.println("failed to generate class for " + fullname + " defined in " + fileParseResult.getContext().getUri());
```

Comment:
can we avoid committing the System.out/err prints? we will handle redirecting logging to stdout/err in a subsequent PR